README.md

```diff
@@ -299,6 +299,29 @@ The ask tab uses the approach programmed in [retrievethenread.py](https://github
 There are also two other /ask approaches with a slightly different approach, but they aren't currently working due to [langchain compatibility issues](https://github.com/Azure-Samples/azure-search-openai-demo/issues/541).
 </details>
 
+<details>
+<summary>How can we view logs from the App Service app?</summary>
+
+You can view production logs in the Portal using either the Log stream or by downloading the default_docker.log file from Advanced tools. 
+
+To log additional things, first set the level of logs you want to see by writing this code inside the `create_app()` function in `app.py`:
+
+```python
+logging.basicConfig(level=logging.INFO)
```

Comment:
We should make this an environment variable

Reply:
ah thats a good idea, so we could always have this line in the code? love it.

Reply:
Done, checked on App Service with multiple levels, updated FAQ.